app/src/main/java/org/oppia/app/mydownloads/MyDownloadsActivity.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.app.mydownloads
+
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The activity for displaying [MyDownloadsFragment]. */
```

Comment:
Can you make this comment more contextual, rather than just displaying fragment.

Reply:
Actually, this activity is not doing anything other than this, meaning it won't handle the logic, it is just handling the fragments.

Can you suggest, what you are expecting?

Reply:
For eg. it contains download list or something similar.

Reply:
Actually, that will be the explanation for the fragment, this activity is not directly handling the download list or update list.

Reply:
Ok got it thanks!